tox/venv.py

```diff
@@ -302,10 +302,14 @@ def run_install_command(self, packages, action, options=()):
                 os.environ.pop('PYTHONPATH')
 
         old_stdout = sys.stdout
-        sys.stdout = codecs.getwriter('utf8')(sys.stdout)
-        self._pcall(argv, cwd=self.envconfig.config.toxinidir, action=action,
-                    redirect=self.session.report.verbosity < 2)
-        sys.stdout = old_stdout
+        try:
+            sys.stdout = codecs.getwriter('utf8')(sys.stdout)
+            self._pcall(argv, cwd=self.envconfig.config.toxinidir, action=action,
+                        redirect=self.session.report.verbosity < 2)
+        except Exception:
+            raise
```

Comment:
can remove these two lines, they do nothing `try: finally:` is valid.

Reply:
changed in https://github.com/tox-dev/tox/pull/724/commits/69e05ace256edecce1f6d57b10f7076ac136472d